docs/getting_started.md

```diff
@@ -190,35 +207,38 @@ scrape_configs:
 
 Go to the expression browser and verify that Prometheus now has information
 about time series that these example endpoints expose, such as the
-`rpc_durations_seconds` metric.
+`node_cpu_seconds_total` (node_exporter) or `wmi_cpu_time_total`
+(wmi_exporter) metric.
 
 ## Configure rules for aggregating scraped data into new time series
 
+If you are using windows you should be able to substitute in
+`wmi_cpu_time_total` for this example.
+
 Though not a problem in our example, queries that aggregate over thousands of
 time series can get slow when computed ad-hoc. To make this more efficient,
 Prometheus allows you to prerecord expressions into completely new persisted
 time series via configured recording rules. Let's say we are interested in
-recording the per-second rate of example RPCs
-(`rpc_durations_seconds_count`) averaged over all instances (but
-preserving the `job` and `service` dimensions) as measured over a window of 5
-minutes. We could write this as:
+recording the per-second rate of cpu time (`node_cpu_seconds_total`) averaged
+over all instances (but preserving the `job`, `instance` and `mode` dimensions)
+as measured over a window of 5 minutes. We could write this as:
 
 ```
-avg(rate(rpc_durations_seconds_count[5m])) by (job, service)
+avg(irate(node_cpu_seconds_total[5m])) by (job, instance, mode)
 ```
 
 Try graphing this expression.
 
 To record the time series resulting from this expression into a new metric
-called `job_service:rpc_durations_seconds_count:avg_rate5m`, create a file
+called `job_instance:node_cpu_usage_ratio:avg_rate5m`, create a file
 with the following recording rule and save it as `prometheus.rules.yml`:
 
 ```
 groups:
-- name: example
+- name: cpu-example
   rules:
-  - record: job_service:rpc_durations_seconds_count:avg_rate5m
-    expr: avg(rate(rpc_durations_seconds_count[5m])) by (job, service)
+  - record: job_instance:node_cpu_usage_ratio:avg_rate5m
+    expr: avg(irate(node_cpu_seconds_total[5m])) by (job, instance, mode)
```

Comment:
This should stay a rate, and mode should not be added.

The convention is also to put the by after the operator.

Reply:
I can change the convention and the function, however adding the `mode` provides the most useful information. Why would we want to provide essentially a straight line for people to look at? If you include mode you can see useful information and provide people with something.

Reply:
As a follow up on convention, does that mean that examples like https://prometheus.io/docs/prometheus/latest/querying/functions/#histogram_quantile should also be updated to follow the `<func> by (<labels>) (<metric>)` convention?

Reply:
If we're using this metric then mode is useful, but we're not really aggregating anything then as you're included the instance label.

Reply:
Valid, then potentially removing the instance? I know for this Quickstart it won't vary but seeing the average utilization across machines. So maybe:
```
- record: job_cluster:node_cpu_usage_ratio:avg_rate5m
  expr: avg by (job, mode) (rate(node_cpu_sections_total[5m]))
```

Reply:
I see this has continued in a different thread. Resolving.